src/Anthropic.Foundry/README.md

```diff
@@ -0,0 +1,72 @@
+# Anthropic.Foundry
+
+Lightweight C# bindings for the Anthropic Foundry surface in the Anthropic SDK for .NET. This project contains the Anthropic.Foundry library used by the solution to interact with Foundry APIs, plus helpers and models used across samples and integrations.
+
+## Contents
+
+- Library source: src/Anthropic.Foundry
+- Unit/integration tests: tests (if present)
+- Samples and usage examples: examples (if present)
```

Comment:
This doesn't seem useful and I would delete (I also don't understand the "if present" thing)